### PR TITLE
Moving setup documentation to GitHub

### PR DESCRIPTION
**Pull request by xpouyat**

### CODE REVIEW

File: doc/set-up-azure-devops.md [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+Local debugging uses user impersonation, and developers sign in with their browsers. To ensure that the application has the correct rights:
+
+1. In Microsoft Entra ID, create a named group such as *Gridwich Devs*, and add the authorized developers to it.
+
+1. In the Azure Subscription, select **Access Control (IAM)** in the left navigation, select **Add role assignments**, and then assign the following roles for **Gridwich Devs**:
+
+   - **Reader and Data Access**
+   - **Storage Blob Data Contributor**
+
+## Next steps
```
Comment (ckittel) [inline doc/set-up-azure-devops.md]: Make sure the VERY NEXT step is the clear call to action.  Each page should lead to the next page.

image

etc.

File: doc/set-up-local-environment.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
+# Gridwich local development environment setup
+
+This article describes how to set up a local Gridwich development environment in either Visual Studio 2019 or above, or Visual Studio Code.
+
+## Prerequisites
+
+- Up-to-date [Visual Studio Code](https://code.visualstudio.com/) or [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/).
+- [Azure CLI](/cli/azure/install-azure-cli)
+- [.NET Core 3.1](https://dotnet.microsoft.com/download/dotnet-core/3.1)
```
Comment (ckittel) [inline doc/set-up-local-environment.md]: Just checking, is this still accurate?

image


You can adjust this AFTER this merges, but want to make sure you're aware.

**Review by ckittel — APPROVED** [review]
Ignoring the maintenance side of this, please just make sure the end of each page leads specifically to the next step (and only that next step).  That way someone can just follow through, click, click, click.  Approving because, in practice, this is a good PR.  Just fix up the next steps and merge :)

File: doc/set-up-local-environment.md (hunk 2 above)
Comment (xpouyat) [reply doc/set-up-local-environment.md]: Good catch, solution uses .NET 6

File: doc/set-up-azure-devops.md (hunk 1 above)
Comment (xpouyat) [reply doc/set-up-azure-devops.md]: Sure

File: doc/set-up-local-environment.md (hunk 2 above)
Comment (ckittel) [reply doc/set-up-local-environment.md]: Right, after this merges, you'll want to do a freshness pass on the content and "catch it up" to the reality of this repo.  This is just one example of the type of thing to do a review for.

**Pull request merged**